docs/Userguide_sharing_data.rst

```diff
@@ -0,0 +1,101 @@
+Sharing Data with ACLs
+======================
```

Comment:
Maybe start by explaining what ACLs are, _and_ provide a link to learn (like https://www.thegeekdiary.com/unix-linux-access-control-lists-acls-basics/? Don't know if it's good, it's just the first Google result 😛)